nsqadmin/static/js/views/lookup.js

```diff
@@ -20,14 +21,23 @@ var LookupView = BaseView.extend({
 
     initialize: function() {
         BaseView.prototype.initialize.apply(this, arguments);
-        $.ajax(AppState.url('/topics?inactive=true')).done(function(data) {
-            this.template = require('./lookup.hbs');
-            this.render({
-                'topics': _.map(data['topics'], function(v, k) {
-                    return {'name': k, 'channels': v};
-                })
-            });
-        }.bind(this));
+        $.ajax(AppState.url('/topics?inactive=true'))
+            .done(function(data) {
+                    this.template = require('./lookup.hbs');
```

Comment:
Let's talk about this separately from error handling, I'm open to (simple) improvements before another React-factoring.